arch/arm64/boot/dts/rockchip/rk3566-radxa-rock-3-compute-module-sodimm.dtsi

```diff
@@ -114,8 +114,8 @@
 
 		sodimm-led {
 			gpios = <&gpio0 RK_PA6 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "timer";
 			pinctrl-0 = <&sodimm_led>;
-			linux,default-trigger = "heartbeat";
```

Comment:
I changed it to align CM3S with the rest of our products that use `heartbeat`. I'd like to see CM3 changed as well, not the other way around.